scripts/docker-entrypoint.sh

```diff
@@ -70,36 +70,66 @@ if [ "$1" = 'mgmtapi' ]; then
 		echo "JVM_OPTS=\"\$JVM_OPTS -javaagent:/etc/cassandra/datastax-mgmtapi-agent-0.1.0-SNAPSHOT.jar\"" >> /etc/cassandra/cassandra-env.sh
 	fi
 
-	CASSANDRA_RPC_ADDRESS='0.0.0.0'
-	CASSANDRA_BROADCAST_RPC_ADDRESS="$(_ip_address)"
-
-    # Not needed as the operator will set all this but leaving for testing
-	for yaml in \
-	  cluster_name \
-		endpoint_snitch \
-		num_tokens \
-		start_rpc \
-		broadcast_address \
-		broadcast_rpc_address \
-		listen_address \
-		rpc_address \
-	; do
-		var="CASSANDRA_${yaml^^}"
-		val="${!var}"
-		if [ "$val" ]; then
-			_sed-in-place "$CASSANDRA_CONF/cassandra.yaml" \
-				-r 's/^(# )?('"$yaml"':).*/\2 '"$val"'/'
-		fi
-	done
-
-	# for rackdc in dc rack; do
-	# 	var="CASSANDRA_${rackdc^^}"
-	# 	val="${!var}"
-	# 	if [ "$val" ]; then
-	# 		_sed-in-place "$CASSANDRA_CONF/cassandra-rackdc.properties" \
-	# 			-r 's/^('"$rackdc"'=).*/\1 '"$val"'/'
-	# 	fi
-	# done
+    # Set this if you want to ignore default env variables, i.e. when running inside an operator
+    if [ $IGNORE_DEFAULTS ]; then
```

Comment:
this needs to be the default way this image works for compatibility with published operator versions